integrations/gnu_libmicrohttpd/CMakeLists.txt

```diff
@@ -14,8 +15,11 @@ add_library(prometheus-http SHARED prometheus_microhttpd.cc)
 
 target_link_libraries(prometheus-http PUBLIC
   ${MHD_LIBRARIES} prometheus-client)
-set_target_properties(prometheus-http PROPERTIES VERSION "0")
-set_target_properties(prometheus-http PROPERTIES SOVERSION "0.0.0")
+set_target_properties(prometheus-http PROPERTIES
+                      VERSION "0"
+                      SOVERSION "0.0.0"
+                      CXX_STANDARD 11
```

Comment:
`CXX_STANDARD` is CMake 3.x? Darn.

We could use `add_target_definitions(...)` instead... or just bump the required version. Not sure which systems don't have 3.x yet...

In any case we have to set the language standard per target and not globally...